medicines/doc-index-updater/Makefile

```diff
@@ -25,6 +25,27 @@ set-env: ## Takes your current .env file and replaces the keyvault value with th
 	--name doc-index-updater-env \
 	--file .env
 
+.PHONY: set-sftp-keys
+set-sftp-keys: ## Adds pub/private key for localhost to your .ssh dir
+	az keyvault secret show \
+		--vault-name mhra-dev \
+		--name dev-sentinel-public-key \
+		--query value \
+		--output tsv > ~/.ssh/doc_index_updater.pub && \
+  az keyvault secret show \
```

Comment:
minor issue and may not be a problem. But Makefiles are tab/space sensitive, so we should probably align